### ISSUE TITLE
[OPR] Only metadata fields that require a human review should be required other are optional. 

### ISSUE THREAD

**Issue by jh-RLI** [OpenEnergyPlatform/oeplatform#1342]
## Description of the issue

We implemented a wee working version of the Open Peer Review. The review process itself is now somewhat stable (stable enough) and we have to trim down the time an average review takes. 

Relevant user interaction:

Less time-consuming
- Accept a field

More time-consuming
- Suggest a field
- Deny a field

Very time-consuming
- Any of the actions above applied on a lot of fields.

## Ideas of solution

Make fields that do not require a human reviewer optional and indicate that to the user.

Only check for required fields to be able to submit or finish a review.

## Workflow checklist
- [ ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/CONTRIBUTING.md)

**Comment by vismayajochem** [comment]
If this should be implemented, a list of "required fields" would be necessary. Maybe it is the same list as in #1248 ?

**Comment by jh-RLI** [comment]
@Ludee Exactly, let’s discuss this. We’re running into a problem here that’s similar to the feedback we’ve frequently received regarding metadata entry in general. There are too many fields. 
This becomes even more apparent during the review process. Currently, we’ve excluded the data model (columns, isAbout, etc.) from the review. Empty fields are currently excluded from the review. All other entries can be reviewed. 

Submitting a review for a complete metadata record is quite tedious, but maybe that’s just the nature of a review. Still, we could perhaps reduce the number of properties a user has to review. We can also introduce mandatory and optional review fields.
